src/components/helpers.tsx

```diff
@@ -0,0 +1,16 @@
+import { css } from "styled-components"
+
+/**
+ * Helper function to display an element as a block that inherits its parents width
+ * @param margin value in pixels to remove from width 100%
+ */
+export const block = (margin: number = 0) => {
+    return props => {
+        if (props.block)
+            return css`
+                display: block;
+                width: calc(100% - ${margin}px);
+                margin: 10px auto;
+            `
+        }
```

Comment:
Thanks y'all 👍 I'm admittedly a little OCD about 4 spaces because it's
like the wider the area I have to read, the less my peripheral "background"
mind can quietly take in from the surrounding code (which impacts focus).
It also seems like it wears the eyes out slightly more over time!